docs/content/middlewares/overview.md

```diff
@@ -148,6 +187,9 @@ then you'll have to append to the middleware name, the `@` separator, followed b
     In this case, since the definition of the middleware is not in kubernetes,
     specifying a "kubernetes namespace" when referring to the resource does not make any sense,
     and therefore this specification would be ignored even if present.
+    On the other hand, if you declare the middleware as a CRD in Kubernetes and use the non-crd Ingress 
+    objects, you'll have to add the kubernetes namespace, the middleware lives in, to the annotation like
```

Comment:
you`ll have to add the kubernetes namespace of the middleware to the annotation... ?

Reply:
Changed with https://github.com/containous/traefik/pull/7269/commits/4143d12d3479b1f7d83b2834fe2234958f994b19